conf/messages.en

```diff
@@ -559,10 +559,12 @@ clientSecondContactHavePhone.error.required = Select yes if they can be contacte
 clientDetailsUpdated.title = Client contact details updated
 clientDetailsUpdated.heading = Client contact details updated
 clientDetailsUpdated.p1 = We will use your client’s new details if we need to contact them.
-clientDetailsUpdated.link = Back to send a CBC report	
+clientDetailsUpdated.link = Back to send a CBC report
 
 clientContactDetailsSaved.title = Client contact details saved
 clientContactDetailsSaved.heading = Client contact details saved
 clientContactDetailsSaved.p1 = We will use your client’s new details if we need to contact them.
 clientContactDetailsSaved.p2 = You can now {0}.
 clientContactDetailsSaved.p2.link = send a CBC report
+
+clientSecondContactHavePhone.error.required = Select yes if we can call the second contact by telephone
```

Comment:
this content should be on line 557, and this line deleted. You have a duplicate message key here